### PR TITLE
feat(orders) ORDERS-4645 Add pickups to storefront order invoice

### PR DESCRIPTION
**Pull request by WillemHoman**
## What?
Add pickups to storefront invoice

## Requirements

- [ ] CHANGELOG.md entry added (required for code changes only)

## Tickets / Documentation

[ORDERS-4645](https://bigcommercecloud.atlassian.net/browse/ORDERS-4645)

[bc-app PR](https://github.com/bigcommerce/bigcommerce/pull/46549)

### Digital only order
Screen Shot 2022-05-26 at 9 57 19 am

### Shipping Single Address
Screen Shot 2022-05-26 at 9 56 46 am

### Pickup Single Address

Screen Shot 2022-05-26 at 11 04 55 am

### Mixed consignment with multiple Shipping addresses 
Screen Shot 2022-05-26 at 9 55 52 am

ping @bigcommerce/orders @bigcommerce/themes-team

### CODE REVIEW

File: templates/pages/account/orders/invoice.html [hunk 1]
```diff
@@ -56,6 +56,23 @@ <h3>{{lang 'account.orders.details.ship_to'}}</h3>
                     </div>
                 {{/unless}}
             {{/if}}
+            {{#if order.pickup_address}}
+                {{#unless order.has_multiple_shipping_addresses}}
```
Comment (donald-nguyen-bc) [inline templates/pages/account/orders/invoice.html:60]: `order.has_multiple_shipping_addresses` means order can have 1 pickup and 1 shipping right?

Comment (WillemHoman) [reply templates/pages/account/orders/invoice.html:60]: yep I used the same variable name for backwards compatibility

File: templates/pages/account/orders/invoice.html [hunk 2]
```diff
@@ -100,12 +117,21 @@ <h2 class="InvoiceHeading">{{lang 'invoice.order_items'}}</h2>
...
                                 {{#each shipping_rows}}
+                                    {{#if is_pickup}}
+                                        <div><b>{{lang 'invoice.pickup'}}</b></div>
+                                    {{/if}}
+                                    {{#if is_shipping}}
+                                        <div><b>{{lang 'invoice.shipping'}}</b></div>
+                                    {{/if}}
+                                    {{#if is_digital}}
+                                        <div><b>{{lang 'invoice.digital'}}</b></div>
+                                    {{/if}}
```
Comment (donald-nguyen-bc) [inline templates/pages/account/orders/invoice.html:132]: 👏 Nice approach, you can work as product designer Willem 😄

Comment (WillemHoman) [reply templates/pages/account/orders/invoice.html:132]: LOL

File: templates/pages/account/orders/invoice.html (hunk 1 above)
Comment (donald-nguyen-bc) [reply templates/pages/account/orders/invoice.html:60]: 👍

**Review by donald-nguyen-bc — APPROVED** [review]
👏 Well done Willem, I will steal this design into the CP invoice template 😄

**Review by bc-vincent-zhao — APPROVED** [review]

File: templates/pages/account/orders/invoice.html [hunk 3]
```diff
@@ -56,6 +56,22 @@ <h3>{{lang 'account.orders.details.ship_to'}}</h3>
...
+                            <li>{{order.pickup_address.line2}}</li>
+                            <li>{{order.pickup_address.city}}, {{order.pickup_address.state}} {{order.pickup_address.zip}}</li>
+                            <li>{{order.pickup_address.country}}</li>
+                            <li>&nbsp;</li>
+                            <li>{{lang 'invoice.phone' number=order.pickup_address.phone}}</li>
+                            {{#if order.pickup_address.email}}<li>{{lang 'invoice.email' email=order.pickup_address.email}}</li>{{/if}}
+                        </ul>
+                    </div>
+                {{/unless}}
+            {{/if}}
```
Comment (bc-munjal) [inline templates/pages/account/orders/invoice.html:74]: 👏

**Review by bc-munjal — APPROVED** [review]

**Comment by bc-munjal** [comment]
🙇 nicely done!!

**Review by BC-krasnoshapka — COMMENTED** [review]
@WillemHoman please don't forget to update CHANGELOG.md

**Comment by WillemHoman** [comment]
> @WillemHoman please don't forget to update CHANGELOG.md

@BC-krasnoshapka thanks for the reminder, is this correct? Should it be in the draft section? https://github.com/bigcommerce/cornerstone/pull/2216/commits/1275d773b3e6c3d936dffe193a6669a2eb47090b

**Comment by WillemHoman** [comment]
@bc-yevhenii-buliuk @bc-vlad-dlogush  could I please get a review on this

**Review by bc-vlad-dlogush — APPROVED** [review]

**Comment by WillemHoman** [comment]
@BC-krasnoshapka @bc-vlad-dlogush @bc-yevhenii-buliuk 
How do I get this merged?
It has been in code review for 3 weeks now and I just had to resolve a conflict.

**Review by BC-krasnoshapka — APPROVED** [review]

**Pull request merged**